packages/components/src/InputEmail/InputEmail.tsx

```diff
@@ -1,24 +1,9 @@
 import React from "react";
-import { FormField, FormFieldProps } from "../FormField";
-
-type InputEmailProps = Pick<
-  FormFieldProps,
-  Exclude<
-    keyof FormFieldProps,
-    | "children"
-    | "autocomplete"
-    | "max"
-    | "min"
-    | "rows"
-    | "type"
-    | "keyboard"
-    | "onEnter"
-    | "onBlur"
-    | "inputRef"
-    | "actionsRef"
-    | "onFocus"
-  >
->;
+import { CommonFormFieldProps, FormField, FormFieldProps } from "../FormField";
+
+interface InputEmailProps
+  extends CommonFormFieldProps,
+    Pick<FormFieldProps, "maxLength" | "readonly" | "validations"> {}
```

Comment:
One i forgot to mention here. We should do this as a type since we are not adding to it.

```suggestion
type InputEmailProps = CommonFormFieldProps &
  Pick<FormFieldProps, "maxLength" | "readonly" | "validations">;
```

Reply:
Fixed here: 5556fa0